config/routes.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+Rails.application.routes.draw do
+  devise_for :users
+  devise_scope :user do  
+    get '/users/sign_out' => 'devise/sessions#destroy'     
+  end
+  resources :categories do
+    resources :transactions
+  end
```

Comment:
- [ ] Please only make use of necessary routes in this project. Do not expose unused routes.

Reply:
- Addressed 😉 👍🏻